tests/telemetry/test_telemetry.py

```diff
@@ -271,7 +273,7 @@ def test_stats_off(monkeypatch):
     mock = Mock()
     posthog_mock = Mock()
     mock.patch(telemetry, '_get_telemetry_info', (False, 'TestUID'))
-    telemetry.log_api("test_action", "ploomber", "0.14.0")
+    telemetry.log_api("test_action", "ploomber", "0.14.0", MOCK_API_KEY)
```

Comment:
Same, not the right place.

Reply:
I am using @edublancas's suggestion. I will make a Telemetry class and put stuff there so we dont have to put all that in every telemetry call